libs/sdk-ui-dashboard/src/model/state/dashboardStore.ts

```diff
@@ -220,6 +224,7 @@ export function createDashboardStore(config: DashboardStoreConfig): ReduxedDashb
             serializableCheck: {
                 ignoredActions: nonSerializableEventsAndCommands,
                 ignoredActionPaths: ["ctx"],
+                ignoredPaths: ["drill.drillableItems"],
```

Comment:
Please add/update a comment on why the check is skipped (I assume because of header predicates which are functions?).